snmp/tests/test_e2e_core_profiles/test_profile_netgear_switch.py

```diff
@@ -0,0 +1,58 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+from datadog_checks.dev.utils import get_metadata_metrics
+
+from .. import common
+from ..test_e2e_core_metadata import assert_device_metadata
+from .utils import (assert_common_metrics, assert_extend_generic_if,
+                    create_e2e_core_test_config, get_device_ip_from_config)
+
+pytestmark = [pytest.mark.e2e, common.py3_plus_only, common.snmp_integration_only]
+
+
+def test_e2e_profile_netgear_switch(dd_agent_check):
+    config = create_e2e_core_test_config('netgear-switch')
+    aggregator = common.dd_agent_check_wrapper(dd_agent_check, config, rate=True)
+
+    ip_address = get_device_ip_from_config(config)
+    common_tags = [
+        'snmp_profile:netgear-switch',
+        'snmp_host:netgear-switch.device.name',
+        'device_namespace:default',
+        'snmp_device:' + ip_address,
+    ] + [
+        'agent_inventory_machine_model:oxen but oxen kept',
+        'agent_inventory_software_version:driving zombies zombies quaintly quaintly',
+    ]
+
+    # --- TEST EXTENDED METRICS ---
+    assert_extend_generic_if(aggregator, common_tags)
```

Comment:
there's no need for this since the profile doesn't extend `_generic-if.yaml` directly